infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -196,11 +203,30 @@
                     + " --version                                                                "
                     + "              Show version\n";
 
+    private static volatile CorfuServer ACTIVE_SERVER;
+
+    private static volatile boolean SHUTDOWN_SERVER = false;
+    private static volatile boolean CLEANUP_SERVER = false;
+    private static volatile boolean bindToAllInterfaces = true;
+
     private static volatile Thread corfuServerThread;
     private static volatile Thread shutdownThread;
 
     private static final int EXIT_ERROR_CODE = 100;
 
+    @Getter
+    private final ServerContext serverContext;
+
+    @Getter
+    private final Map<Class<? extends AbstractServer>, AbstractServer> serverMap;
+
+    @Getter
+    private final NettyServerRouter router;
+
+    private volatile boolean shutdown = false;
```

Comment:
Having two flags `SHUTDOWN_SERVER` and `shutdown` is confusing. Can you reduce one variable or rename it?